homu/main.py

```diff
@@ -1064,6 +1064,8 @@ def arguments():
                     'continuous integration service')
     parser.add_argument('-v', '--verbose',
                         action='store_true', help='Enable more verbose logging')
+    parser.add_argument('-c', '--config',
+                        action='store', help='Path to cfg.toml', default='cfg.toml')
```

Comment:
I already am differentiating via the `if args.config == 'config.toml'`.  I tried reworking this but it ended up messier, I'm not sure how much we really care about the JSON path.

```
diff --git a/homu/main.py b/homu/main.py
index 173ac8f..0904dbd 100644
--- a/homu/main.py
+++ b/homu/main.py
@@ -1065,7 +1065,7 @@ def arguments():
     parser.add_argument('-v', '--verbose',
                         action='store_true', help='Enable more verbose logging')
     parser.add_argument('-c', '--config',
-                        action='store', help='Path to cfg.toml', default='cfg.toml')
+                        action='store', help='Path to cfg.toml')
 
     return parser.parse_args()
 
@@ -1077,16 +1077,24 @@ def main():
     logger.setLevel(logging.DEBUG if args.verbose else logging.INFO)
     logger.addHandler(logging.StreamHandler())
 
-    try:
-        with open(args.config) as fp:
-            cfg = toml.loads(fp.read())
-    except FileNotFoundError:
-        # Fall back to cfg.json only if we're using the defaults
-        if args.config == 'cfg.toml':
-            with open('cfg.json') as fp:
+    if args.config is None:
+        config_paths = ['cfg.toml', 'cfg.json']
+    else:
+        config_paths = [args.config]
+
+    cfg = None
+    for path in config_paths:
+        if not os.path.isfile(path):
+            continue
+        if path.endswith('.json'):
+            with open(path) as fp:
                 cfg = json.loads(fp.read())
         else:
-            raise
+            with open(path) as fp:
+                cfg = toml.loads(fp.read())
+    if cfg is None:
+       logger.error("Failed to find config file in paths: {}", config_paths) 
+       sys.exit(1)
 
     gh = github3.login(token=cfg['github']['access_token'])
     user = gh.user()
```